cmd/test.go

```diff
@@ -0,0 +1,182 @@
+package cmd
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"path"
+	"time"
+
+	"github.com/charmbracelet/log"
+	"github.com/omnigres/cli/orb"
+	"github.com/spf13/cobra"
+)
+
+var testCmd = &cobra.Command{
+	Use:   "test",
+	Short: "Test orbs",
+	Run: func(cmd *cobra.Command, args []string) {
+		var cluster orb.OrbCluster
+		var err error
+		cluster, err = getOrbCluster()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		log.Debug("Workspace", "workspace", workspace)
+		cwd, err := os.Getwd()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		orbs, err := currentOrbs(cluster, cwd)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		ctx := context.Background()
+		log.Debug("Assembling orbs src", "orbs", orbs)
+
+		t := time.Now().Format("20060102150405")
+		nameForTestDatabase := func(orbName string) string {
+			return fmt.Sprintf("%s_%s_%s", orbName, "test", t)
+		}
+
+		err = assembleOrbs(ctx, cluster, true, orbs, nameForTestDatabase)
+		err = testOrbs(ctx, cluster, orbs, nameForTestDatabase)
+
+		if err != nil {
+			log.Fatal(err)
+		}
+	},
+}
+
+func testOrbs(
+	ctx context.Context,
+	cluster orb.OrbCluster,
+	orbs []string,
+	databaseForOrb func(string) string,
+) (err error) {
+
+	var testTarget, testRunner *sql.DB
+	testRunner, err = cluster.Connect(ctx, "omnigres")
+	if err != nil {
+		return
+	}
+
+	testOrb := func(orbName string) error {
+		dbName := databaseForOrb(orbName)
+		testTarget, err = cluster.Connect(ctx, dbName)
+		if err != nil {
+			return err
+		}
+		log.Debug("Testing orb", "orbName", orbName, "dbName", dbName)
+
+		_, err = testRunner.ExecContext(
```

Comment:
May be we should add support for test database preparation and teardown to omni_test? We can't migrate for it (CLI's job), but we can tuck away the `template` stuff as an implementation detail.